apstools/devices/aps_machine.py

```diff
@@ -88,12 +88,18 @@ class ApsMachineParametersDevice(Device):
     # 'Stored Beam',
     # 'Delivered Beam',
     # 'MAINTENANCE')
-    shutter_permit = Component(EpicsSignalRO, "ACIS:ShutterPermit", string=True)
+
+    # shutter_permit = Component(EpicsSignalRO, "ACIS:ShutterPermit", string=True) # Not Found
```

Comment:
I wonder if I can find this PV (or its replacement) on one of the other screens...

Reply:
A beamline-independent PV is not obvious.  Another way is to do a recursive grep through the screen files, such as:

    grep -R ShutterPermit . | grep ACIS

Reply:
./asdops/screens/SRcomfort.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/SRcomfort.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/SRcomfort.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/checklist.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/SRInj24s.adl:          chan="ACIS:ShutterPermit"
./asdops/screens/SRInjHybrid.adl:               chan="ACIS:ShutterPermit"
./asdops/screens/SRcomfort_2015.adl:            chan="ACIS:ShutterPermit"
./asdops/screens/SRcomfort_2015.adl:            chan="ACIS:ShutterPermit"
./asdops/screens/SRInj324s.adl:         chan="ACIS:ShutterPermit"
./asdops/screens/SRInj24s_P0.adl:               chan="ACIS:ShutterPermit"
./asdops/screens/SRInj324s_P0.adl:              chan="ACIS:ShutterPermit"
./asdops/screens/SRInjHybrid_P0.adl:            chan="ACIS:ShutterPermit"
```

Yet, this PV does not respond to `caget`